openfisca_core/tracers.py

```diff
@@ -102,7 +103,12 @@ def record_calculation_end(self, variable_name, period, result, **parameters):
                 u"Something went wrong with the simulation tracer: result of '{0}' was expected, got results for '{1}' instead. This does not make sense as the last variable we started computing was '{0}'."
                 .format(expected_key, key).encode('utf-8')
                 )
-        self.trace[key]['value'] = result.tolist()  # Cast numpy array into a python list
+        intermediate_result = result.tolist()  # Cast numpy array into a python list
```

Comment:
Hungry for more information about the `intermediate_result` and a more specific name might help. ^^